lib/backend/etcdbk/etcd.go

```diff
@@ -771,11 +833,16 @@ func (b *EtcdBackend) Get(ctx context.Context, key []byte) (*backend.Item, error
 		return nil, trace.NotFound("item %q is not found", string(key))
 	}
 	kv := re.Kvs[0]
-	bytes, err := unmarshal(kv.Value)
+	value, err := unmarshal(kv.Value)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	return &backend.Item{Key: key, Value: bytes, ID: kv.ModRevision, LeaseID: kv.Lease}, nil
+	return &backend.Item{
+		Key:      key,
+		Value:    value,
+		ID:       kv.ModRevision,
+		Revision: toBackendRevision(re.Header.Revision),
```

Comment:
If I'm reading the docs right, the `Revision` in the `ResponseHeader` for a range request is the revision number at the point where the request was processed, not the revision number for when the item was last modified (we expect different `ModRevision`s for different items in a `GetRange`, after all).

```suggestion
		Revision: toBackendRevision(kv.ModRevision),
```